test/end-to-end/auth-client.js

```diff
@@ -52,3 +52,11 @@ test('exchangeRefreshToken', async(t) => {
     'refreshToken',
   ]), []);
 });
+
+test('compatibility', async(t) => {
+  const client = new smartcar.AuthClient(getAuthClientParams());
+
+  const comp = client.compatibility('5YJXCDE22HF068739');
```

Comment:
can you store the vin in a variable that describes the car, something like  `TESLA_MODEL_S` or something of the like